scaffolds/app_engine/app/server.py

```diff
@@ -0,0 +1,35 @@
+""" Flask serving API and small demo UI.
+"""
+
+import logging
+
+from flask import Flask, jsonify, request, send_file
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def _index():
+    """Serve index.html in the static directory"""
+    return send_file("static/index.html")
+
+
+@app.route("/myapp", methods=["GET"])
+def _answernaut():
```

Comment:
Maybe remove this internal code name and use more generic one?

Reply:
I'll do both! Thanks @takumiohym .